HUBER/iocBoot/iocHUBER-IOC-01/config.xml

```diff
@@ -4,8 +4,12 @@
 <ioc_desc>Huber Controller</ioc_desc>
 <ioc_details></ioc_details>
 <macros>
+<macro name="PORT" pattern="^COM[0-9]+$" description="Serial COM port" hasDefault="NO" />
+<macro name="EMULATOR_PORT" pattern="^[0-9]+$" description="IP address of Huber controller" defaultValue="57677" hasDefault="YES" />
```

Comment:
Emulator port? I don't think we want to expose this to users. You can set macros without them being in config.xml - the config.xml ones are just the ones that show up in the GUI.